docs/docs/guides/motion_detection.md

```diff
@@ -0,0 +1,64 @@
+---
+id: motion_detection
+title: Tuning Motion Detection
+---
+
+# Tuning Motion Detection
+
+Frigate uses motion detection as a first line check to see if there is anything happening in the frame worth checking with object detection.
+
+Once motion is detected, it tries to group up nearby areas of motion together in hopes of identifying a rectangle in the image that will capture the area worth inspecting. These are the red "motion boxes" you see in the debug viewer.
+
+## The Goal
+
+The default motion settings should work well for the majority of cameras, however there are cases where tuning motion detection can lead to better and more optimal results. Each camera has its own environment with different variables that affect motion, this means that the same motion settings will not fit all of your cameras.
+
```

Comment:
I'm still not clear on the goal of motion detection.

- It can't be to prevent detection when there is no motion, because frigate continues to track objects that stop moving.
- It can't be to maximize detector framerate, because giving the detector possibly several 320x320 input tensors per frame is a larger burden than giving it one 320x320 input tensor for the whole frame.
- It can't be to reduce CPU, because motion detection is more expensive than just resizing the whole image to pass to the detector

Is it to reduce average power draw? That would assume that the average detect fps falls sufficiently below 5 to make up for the extra power required for motion detection. Is that typical?

Is it to improve detector performance by getting the object to fill more of the input tensor? I wouldn't naturally assume that that helps with recall, but I could be convinced if there's evidence.

Is it to reduce false positives?

Reply:
> It can't be to prevent detection when there is no motion, because frigate continues to track objects that stop moving.

I don't see how you reach this conclusion. Motion detection means frigate only looks for objects once motion is detected. 

The reason frigate continues tracking objects when they stop moving is being it knows where to look for them due to the object tracker. 

Also, object detection results are far worse sending the entire frame versus cropping and running on regions. Also, while you could train a model, you'd basically need a different model for every camera.

Reply:
Motion detection is also used for recordings when users only want motion based 24/7 recordings